src/asmcnc/apps/systemTools_app/screens/xy_jig/screen_xy_jig_decision.py

```diff
@@ -0,0 +1,92 @@
+from kivy.uix.screenmanager import Screen
+from kivy.lang import Builder
+
+from asmcnc.skavaUI import popup_info
+
+from asmcnc.apps.systemTools_app.screens.xy_jig import screen_xy_jig
+from asmcnc.apps.systemTools_app.screens.xy_jig import screen_xy_jig_monitor
+from asmcnc.apps.systemTools_app.screens.xy_jig import screen_xy_jig_manual_move
+
+from asmcnc.comms.yeti_grbl_protocol.c_defines import *
+
+Builder.load_string("""
+<XYJigDecision>:
+
+    BoxLayout:
+        padding: [dp(20), dp(150)]
+        spacing: dp(20)
+
+        Button:
+            text: 'X Single Stack'
+            font_size: dp(50)
+            bold: True
+            text_size: self.size
+            halign: 'center'
+            valign: 'middle'
+            background_color: [1,0,0,1]
+            background_normal: ''
+            on_press: root.enter_xy_jig('X_single')
+
+        Button:
+            text: 'X Double Stack'
+            font_size: dp(50)
+            bold: True
+            text_size: self.size
+            halign: 'center'
+            valign: 'middle'
+            background_color: [1,0,0,1]
+            background_normal: ''
+            on_press: root.enter_xy_jig('X_double')
+
+        Button:
+            text: 'Y'
+            font_size: dp(50)
+            bold: True
+            text_size: self.size
+            halign: 'center'
+            valign: 'middle'
+            background_color: hex('#00C300FF')
+            background_normal: ''
+            on_press: root.enter_xy_jig('Y')
+
+""")
+
+
+class XYJigDecision(Screen):
+
+    def __init__(self, **kwargs):
+        super(XYJigDecision, self).__init__(**kwargs)
+
+        self.systemtools_sm = kwargs['systemtools']
+        self.m = kwargs['m']
+        self.l = kwargs['l']
+
+    def enter_xy_jig(self, axis):
```

Comment:
it might also be worth having a check that the chosen single/double stack motor option matches up with the current TMC registers (i.e. double stacks should have a current of ~20 when someone enters this screen), just in case someone presses the wrong button and then blows up a single stack motor. It can be an "are you sure?" that still allows them to continue if currents don't match (rather than a full block)

Reply:
What are the ranges that I should allow when doing this?